### ISSUE TITLE
Grond truth calculation

### ISSUE THREAD

**Issue by rafikg** [Stick-To/CenterNet-tensorflow#10]
Hello @Stick-To ,
Did you calculate these ground truth quantities "Heat map, offset, size" and the focal loss? I think you did only a simulation of training ? Thanks

**Comment by Stick-To** [comment]
I haven't not calculated the ground truth quantities, I focus on the reimplemention

**Comment by rafikg** [comment]
Did you load the weights of the DLA-34 of the original repo (without considering the Deformable conv)?
Based on your results, is it important to include the Deformable conv, because I think it very computational if we use high level keras implementation. Thanks

**Comment by Stick-To** [comment]
No, I pretrained it several epochs. If you want to load weights from the original repo, you could change the code

**Comment by rafikg** [comment]
Thanks for your time, but I can't see if you did not calculate the ground truth, how do you do the training?

**Comment by Stick-To** [comment]
Oh, Sorry, I have misunderstood your problem. 
offset gt  row 198
size gt  row 199
heatmap gt row 227-245
focal loss   row 248-250
